gsa/src/web/pages/tasks/detailspage.js

```diff
@@ -344,6 +347,37 @@ const Page = ({
   const deleteTask = dTask =>
     del(dTask.id).then(goto_list('tasks', {history})).catch(onError);
 
+  const gmpSettings = useGmpSettings();
+
+  const timeoutFunc = useCallback(
+    ({isVisible}) => {
+      if (!isVisible) {
+        return gmpSettings.reloadIntervalInactive;
+      }
+      if (task.isActive()) {
+        return gmpSettings.reloadIntervalActive;
+      }
+      return gmpSettings.reloadInterval;
+    },
+    [task, gmpSettings],
+  );
+
+  const reloadFunc = useCallback(() => refetch(), [refetch]);
+  const [startReload, stopReload, hasRunningTimer] = useReload(
+    reloadFunc,
+    timeoutFunc,
+  );
```

Comment:
Not sure if I did that wrong at the list page too but there is no need to define a reloadFunc here. We can use refetch directly.
```suggestion
  const [startReload, stopReload, hasRunningTimer] = useReload(
    refetch,
    timeoutFunc,
  );
```

Reply:
Yeah, I got that from the listpage. I am going to adjust both pages.